src/csv2docx/cli.py

```diff
@@ -21,4 +21,9 @@
 )
 @click.option("--path", "-p", default="output", help="The location to store the files.")
 def main(data: str, template: str, name: str, path: str, delimiter: str) -> None:
-    csv2docx.convert(data, template, name, path, delimiter)
+    try:
+        csv2docx.convert(data, template, name, path, delimiter)
+    except IOError:
+        click.echo(f"The file {path} could not be opened or found.")
+    except (KeyError, ValueError) as err:
+        click.echo(err)
```

Comment:
I would instead change `click.echo` to `click.ClickException` and test that the exceptions are thrown. At the moment, testing for string comparisons is not ideal as: (i) does not indicate failure; and (ii) the content of the string may change? 

e.g., `click.echo(err)` could become `raise click.ClickException(err)`

Reply:
Good point - I've implemented the ClickException, and have removed the string comparison as - you indicate - this might change.